src/octave/nlopt_optimize_usage.h

```diff
@@ -0,0 +1,192 @@
+#define NLOPT_OPTIMIZE_USAGE \
```

Comment:
This file is supposed to be generated by `cmake`, no?

Reply:
It was not for me, I had to add it manually.